pkg/gui/keybindings.go

```diff
@@ -30,15 +32,207 @@ func (b *Binding) GetKey() string {
 	case rune:
 		key = int(b.Key.(rune))
 	case gocui.Key:
+		if b.Key.(gocui.Key) == gocui.KeyCtrlSpace {
```

Comment:
what are your thoughts on having these mappings in a map i.e.
```
		keyMap := map[gocui.Key]string{
			gocui.KeyCtrlSpace:     "ctrl+space",
			gocui.KeyCtrlBackslash: "ctrl+\\",
			...
		}

		value, ok := keyMap[b.Key.(gocui.Key)]
		if ok {
			return value
		}
```

Reply:
That's a good idea. Some of the keybindings were duplicated anyway (for example `esc` and `ctrl+[`).